src/pcapi/sandboxes/scripts/creators/industrial/create_industrial_webapp_users.py

```diff
@@ -27,7 +27,15 @@
 
 
 def create_industrial_webapp_users():
-    logger.info("create_industrial_webapp_users")
+    young_users = create_industrial_webapp_young_users()
```

Comment:
```suggestion
    beneficiaries_users = create_industrial_webapp_beneficiaries()
```
Si on parle bien des bénéficiaires du pass (ceux qui ont du crédit), alors c'est le terme métier, pas "jeune" qui est trop imprécis

Reply:
👍

Reply:
je vais renommer en Jeunes et GrandPublic ça a plus de sens en terme métier